front/app/components/admin/GraphCard/index.tsx

```diff
@@ -17,15 +17,17 @@ interface Props {
   infoTooltipContent?: React.ReactChild;
   exportMenu?: ReportExportMenuProps;
   children?: React.ReactNode;
+  fullWidth?: boolean;
 }
 
 const GraphCard = ({
   title,
   infoTooltipContent,
   exportMenu,
   children,
+  fullWidth = true,
 }: Props) => (
-  <GraphCardContainer className="fullWidth dynamicHeight">
+  <GraphCardContainer className={`dynamicHeight ${fullWidth && 'fullWidth'}`}>
```

Comment:
```suggestion
  <GraphCardContainer className={`dynamicHeight ${fullWidth ? 'fullWidth' : ''}`}>
```

`fullWidth && 'fullWidth'` evaluates to `false` if `fullWidth` is `false`, so you would get `false` as a class name in that case